### DISCUSSION TITLE
Add a sniffing network interface

### DISCUSSION THREAD

**Discussion by ricamz** [Security-Onion-Solutions/securityonion#11794] [General] [answered]
Hi there.

I am running S.O. 2.3.270 standalone in a VMWare virtual machine.

I want to add a network sniffing interface, which I didn't do in the initial install.

I have a bond0 and eth0 (management) interfaces, but don't see any traffic on bond0.

The virtual machine have two interfaces.

Can you guide me in setting up network visibility?

RS

**Comment by robbiemarshall** [comment, marked as answer]
You'll want to use so-monitor-add to add the additional sniffing interface to bond0.

**Comment by ricamz** [comment]
Thank you @robbiemarshall .

So the new interface should appear in the linux and that interface should be added with so-monitor-add.

Thank you fir your help. Will test and return if I have any problem.
